src/20_Components/amp-date-picker.html

```diff
@@ -0,0 +1,202 @@
+      </template>
+    </amp-date-picker>
+  </div>
+  <!--
+    In addition, you can also use `amp-mustache` to create templates for custom date markup.
+    Here we are using a taco icon in place of the number of the month with a frequency of 2 weeks every Tuesday.
+  -->
+  <div>
+    <button class="ampstart-btn m1 caps" on="tap:simple-date-picker.clear">Clear</button>
+    <amp-date-picker
+      type="single"
+      layout="container"
+      width="300"
+      height="300"
+      on="select:AMP.setState({date: event.date, dateType: event.id})"
+      locale="en"
+      format="Y-MM-DD"
+      number-of-months="1"
+      show-clear-date
+      keep-open-on-date-select
+      id="simple-date-picker-2"
+      class="mr1 ml1">
+      <input amp-date-placeholder placeholder="Pick a date!!"/>
+      <template
+        type="amp-mustache"
+        date-template
+        dates="FREQ=WEEKLY;DTSTART=20170101T160000Z;INTERVAL=2;WKST=SU;BYDAY=TU 2017-10-30"
+        id="tacos1">
+        <span>🌮</span>
+        <span class="taco-tuesday"></span>
+      </template>
+      <template type="amp-mustache" info-template>
+        <span
+          [text]="'You picked ' + date + '.' +
+          (dateType == 'tacos1' ? ' Happy Taco Tuesday!' : '')"
+        >You will see your chosen date here.</span>
+      </template>
+    </amp-date-picker>
+  </div>
+  <!--
+    `amp-date-picker` supports a range of attributes, for example `highlighted` and `blocked`. Find the complete list int the official [doc](https://github.com/ampproject/amphtml/tree/master/extensions/amp-date-picker/0.1).
+    The `blocked` attribute allows to specify a space separated list of ISO 8601 dates and RFC 5545 RRULEs specifying disallowed dates.
+    The `highlighted` attributes a space separated list of ISO 8601 dates and RFC 5545 RRULEs specifying dates displayed with a highlight style.
+  -->
+  <div>
+    <button class="ampstart-btn m1 caps" on="tap:simple-date-picker.clear">Clear</button>
+    <amp-date-picker
+      type="single"
+      layout="container"
+      width="300"
+      height="300"
+      on="select:AMP.setState({date: event.date, dateType: event.id})"
+      locale="en"
+      format="Y-MM-DD"
+      number-of-months="1"
+      show-clear-date
+      keep-open-on-date-select
+      highlighted="2017-10-10 2017-10-12"
+      blocked="2017-10-10 2017-10-17 2017-10-19 2017-10-21"
+      id="simple-date-picker-3"
+      class="mr1 ml1">
+      <input amp-date-placeholder placeholder="Pick a date!!"/>
+      <template type="amp-mustache" info-template>
+        <span [text]="date != null ? 'You picked ' + date + '.' : 'You will see your chosen date here.'">You will see your chosen date here.</span>
+      </template>
+    </amp-date-picker>
+  </div>
+  <!-- #### Date Range Picker -->
+  <!--
+    `amp-date-picker` with `type=range` can be used for selecting a date range. The default css will disable
```

Comment:
Does the css really disables this? Suggestion: "By default, amp-date-picker disables past dates, use ..."

Reply:
Yes it does.